docs/settings/cases-settings.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[cases-settings]]
+== Cases settings in {kib}
+++++
+<titleabbrev>Cases settings</titleabbrev>
+++++
+
+You do not need to configure any additional settings to use <<cases,cases>> in {kib}.
+To provide greater control over case features, you can configure the following settings in the `kibana.yml` file:
+
+`xpack.cases.files.allowedMimeTypes`::
+The MIME types that you can attach to a case, represented in an array of strings.
+The default MIME types are specified in https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/cases/common/constants/mime_types.ts[mime_types.ts].
+
+`xpack.cases.files.maxSize`::
+The file size limit for files that you can attach to a case, represented as the number of bytes. Defaults to 10 MiB for images and 100 MiB for all other MIME types.
```

Comment:
IMO this text is unclear on whether the number value is in bytes or MiB. If the former, I'd suggest changing to something like this:
```suggestion
The file size limit for files that you can attach to a case, represented as the number of bytes. Defaults to `10485760`, which means a maximum of 10 MiB for images and 100 MiB for all other MIME types.
```